app/lib/search/index_simple.dart

```diff
@@ -348,6 +366,9 @@ Set<String> _tokenize(String originalText) {
       prevLower = lower;
     }
   }
+  if (minLength > 0) {
+    tokens.removeWhere((t) => t.length < minLength);
```

Comment:
we do have that for n-grams (`max(minNgram, minLength)`), and prefix is also starting at maxNgram+1, but we add the word, and also the parts of the CamelCasedWords, and instead of putting the check everywhere, it seemed to be easier to just remove them if they happen to be shorter.